docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-normalize-data-model/index.md

```diff
@@ -54,7 +84,7 @@ For each `event_names` listed, a model is generated for records with matching ev
 For example, if you have 3 `event_names` listed as `['page_view']`, `['page_ping']`, and `['link_click', 'deep_link_click']` then 3 models will be generated, each containing only those respective events from the atomic events table.
 
 ### Filtered Events Model
-A single model is built that provides `event_id`, `collector_tstamp` and the name of the Normalized Event Model that the event was processed into, it does not include records for events that were not of an event type in your configuration. The model file itself is a series of `UNION` statements.
+A single model is built that provides `event_id`, `snowplow__partition_tstamp` (defaults to `collector_tstamp`), and the name of the Normalized Event Model that the event was processed into, it does not include records for events that were not of an event type in your configuration. The model file itself is a series of `UNION` statements.
```

Comment:
If they change it, it will still provide collector_tsamp, right?

Reply:
Dont get it

Reply:
This is hardcoded as collector_tstamp still right?

Reply:
Actually for filtered events its dynamic. For the rest is the fixed + the variable

Reply:
Oh yes, sorry, it was the users model in the end that we did not change. Cool.